eclipse-scout-core/src/encoder/PlainTextEncoder.ts

```diff
@@ -48,22 +48,29 @@ export class PlainTextEncoder {
     // It is not possible to use jquery's text() function or to create a html element and use textContent, because the new lines get omitted.
     // Node.innerText would preserve the new lines but it is not supported by firefox
 
+    // Remove comments
+    text = text.replace(/<!--.*?--!?>/gs, '');
+
+    // Remove font icons (needs to be executed before removing attribute values)
+    if (options.removeFontIcons) {
+      text = text.replace(/<span\s[^>]*class="[^"]*font-icon[^"]*"[^>]*>[^<]*<\/span>/gmi, '');
```

Comment:
I know that the old code did this too, but is the "m" flag really necessary here? There are no ^ or $ anchors in this regex.